src/app/components/dialogs/mapSettingsDialog/mapSettingsDialog.component.html

```diff
@@ -0,0 +1,34 @@
+<app-base-dialog [data]="data">
+  <mat-drawer-container class="example-container" autosize>
+    <mat-drawer #drawer [opened]="true" mode="side">
+      <mat-selection-list #optionList [multiple]="false">
+        <mat-list-option [selected]="true" value="general">General</mat-list-option>
+        <mat-list-option value="advanced">Advanced</mat-list-option>
+      </mat-selection-list>
+    </mat-drawer>
+
+    <form class="example-sidenav-content" [formGroup]="settingsForm">
+      <div *ngIf="optionList.selectedOptions.selected[0]?.value === 'general'" class="color-box">
+        <p>General</p>
+        <mat-selection-list #colourGradientList formControlName="generalColourSelection"
+          [(ngModel)]="generalSelectionValue" [multiple]="false">
+          <!-- [selected]="colourGradientItem.colourGradient.value === currentSelection" -->
+          <mat-list-option *ngFor="let colourGradientItem of gradientList" [value]="colourGradientItem.colourGradient">
+            <span class="colour-option">{{colourGradientItem.name}} <span
+                [ngClass]="['colour-sample', colourGradientItem.colourGradient]"></span></span>
+          </mat-list-option>
+        </mat-selection-list>
+      </div>
+      <div *ngIf="optionList.selectedOptions.selected[0]?.value === 'advanced'" class="color-box"
+        [ngStyle]="{'background-color': '#2ca25f'}">
+        <p>Advanced</p>
+      </div>
+      <div class="button-wrapper">
+        <button mat-raised-button [disabled]="settingsForm.pristine" (click)="applyChanges()">Apply</button>
```

Comment:
Clicking apply doesn't actually apply, and clicking close does.  Maybe just have one button which is "apply changes" which applies and closes?